pyproject.toml

```diff
@@ -20,11 +20,12 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Chemistry",
 ]
 dependencies = [
-    "numpy>=1.24.3",
-    "scipy>=1.10.1",
-    "matplotlib>=3.7.1",
-    "pandas>=2.0.2",
-    "attrs>=23.1.0",
+    # please don't increase these version numbers without a reason.
```

Comment:
Commonly they would be suggested to be increased by dependabot, due to critical or security bugs. Is this packaged being used in a setting, where packages cannot easily be upgraded. If "yes" I would like to know which settings, so I know what forms the limit for upgrading.

Reply:
I just know that pip could not auto-upgrade the packages to the required versions, preventing me from installing it on my own computer... and that it works witht the versions I have.

Reply:
This is puzzling. Could you try with a fresh environment. The tests run (and install from that requirement list) in CI on python versions 3.8 -> 3.11, so I really don't see any reason why should be a problem.

Reply:
Works now, and I've reverted the version numbers! See top-level comment below.